src/photos.cs

```diff
@@ -1140,16 +1194,289 @@ interface PHLivePhotoFrame {
 		nfloat RenderScale { get; }
 	}
 
-	// TODO returns null on Xcode 8 beta 1-6, rdar #28169810 https://trello.com/c/RwXK6YRX
-	// PHLivePhotoEditingContext.PrepareLivePhotoForPlayback and SaveLivePhoto could use a strong dictionary once it's available
-#if false
-	[iOS (10,0)]
-	[TV (10,0)]
+#if XAMCORE_2_0 // fails to build with mac/classic
```

Comment:
@dalexsoto I recall we discussed that a while ago, was a bug filled ? (could not find it)